mobile/src/main/java/org/openhab/habdroid/ui/OpenHABBindingFragment.java

```diff
@@ -209,17 +205,17 @@ public void  onFailure(int statusCode, Header[] headers, byte[] responseBody, Th
     }
 
     private void stopProgressIndicator() {
-        if (mActivity != null)
```

Comment:
If statements without curly braces only apply to the next one line. So this null check applied to only the log statement and the following call on "mActivity" would still crash if it was null.